modules/custom/az_person/az_person_profile_import/migrations/az_person_profile_import.yml

```diff
@@ -0,0 +1,76 @@
+id: az_person_profile_import
+label: Profiles Integration
+migration_tags:
+  - Profiles Integration
+source:
+  plugin: url
+  data_fetcher_plugin: http
+  data_parser_plugin: az_person_profile_import_json
+  urls: []
+  item_selector: /
+  ids:
+    netid:
+      type: string
+  constants:
+    NAME: 'name'
+
+  fields:
+    -
+      name: netid
+      selector: 'Person/netid'
+    -
+      name: surname
+      selector: 'Person/surname'
+    -
+      name: givenname
+      selector: 'Person/givenname'
+    -
+      name: email
+      selector: 'Person/email'
+    -
+      name: biography
+      selector: 'Bio/desc'
+    -
+      name: titles
+      selector: 'Titles'
+    -
+      name: phone
+      selector: 'Person/phone'
+    -
+      name: degrees
+      selector: 'Degrees'
+
+process:
+  type:
+    plugin: default_value
+    default_value: az_person
```

Comment:
```suggestion
  type:
    plugin: default_value
    default_value: az_person
  revision_log:
    plugin: default_value
    default_value: 'Imported from Profiles'
```

Reply:
I think we should mention the specific fields it is updating.

Reply:
> Do you think this is a good idea?

Migrate imports don't seem to currently create revisions: [Add option to create new revision in entity migration](https://www.drupal.org/project/drupal/issues/2988382). We are currently forcing this to happen in `hook_entity_presave()`

Reply:
> I think we should mention the specific fields it is updating.

@trackleft Do you think that the revision log message should mention exactly which fields were changed?